Website/Website/config.js

```diff
@@ -12,12 +12,12 @@ function initializeLocalServices() {
     Object.keys(webComposition.localServices).forEach(key => {
```

Comment:
Before doing Object.keys on webComposition.localServices, can you add a if check to check if webComposition.localServices is defined before using it in Object.keys

i.e.
if (webComposition.localServices) {
    Object.keys(...)
    ...
}

Reply:
Done